### DISCUSSION TITLE
[疑似BUG]在自定义主页中无法添加带有=号的URL

### DISCUSSION THREAD

**Discussion by LUNACYGG** [Meloong-Git/PCL#150] [提问] [answered]
在我试图绑定一个按钮并打开我们服务器的加群URL时，PCL出现了报错，原因是"="是意外的标记，我不确定这是不是我个人问题，希望得到解决

我对此的解决方案：将EventType更改为打开文件并将文件设置为一个指向加群URL的快捷方式

**Comment by LTCatt** [comment, marked as answer]
需要自己进行转义，详情百度 XAML 转义
